src/current/v24.3/create-trigger.md

```diff
@@ -0,0 +1,190 @@
+---
+title: CREATE TRIGGER
+summary: The CREATE TRIGGER statement creates a trigger on a specified table.
+toc: true
+keywords:
+docs_area: reference.sql
+---
+
+The `CREATE TRIGGER` [statement]({% link {{ page.version.version }}/sql-statements.md %}) defines a [trigger]({% link {{ page.version.version }}/triggers.md %}) on a specified table.
```

Comment:
On the sql-statements.md page `CREATE TRIGGER` is not listed. Should it be listed?

Reply:
Great catch, adding this and DROP TRIGGER